### ISSUE TITLE
error reading file: expected 'vorbis' identification type 1 with Ogg Opus file

### ISSUE THREAD

**Issue by deluan** [dhowden/tag#72]
Hey guys, I cannot open the Ogg Opus file attached, seems that it contains an invalid vorbis comment. This is what I get when running `tag file.ogg`:
```sh
$ tag file.opus
error reading file: expected 'vorbis' identification type 1
```

Note that `ffmpeg` is able to extract all metadata _and_ the embedded picture.

[file.opus.zip](https://github.com/dhowden/tag/files/4695531/file.opus.zip)

**Comment by wader** [comment]
Without #69? i get this with that change:

```sh
$ go run cmd/tag/main.go file.opus
Metadata Format: VORBIS
File Type: OGG
 Title: Some Might Say
 Album: Some Might Say
 Artist: Oasis
 Composer:
 Genre: Alternative
 Year: 2014
 Track: 1 of 4
 Disc: 1 of 1
 Picture: Picture{Ext: jpg, MIMEType: image/jpeg, Type: Cover (front), Description: , Data.Size: 704476}
 Lyrics: Some might say that sunshine follows thunder
Go and tell it to the man who cannot shine
Some might say that we should never ponder
On our thoughts today 'cause they hold sway over time

Some might say we will find a brighter day
Some might say we will find a brighter day, yeah

'Cause I've been standing at the station
In need of education in the rain
You made no preparation for my reputation once again
The sink is full of fishes
She's got dirty dishes on the brain
It was overflowing gently but it's all elementary my friend

Some might say they don't believe in heaven
Go and tell it to the man who lives in hell
Some might say you get what you've been given
If you don't get yours I won't get mine as well

Some might say we will find a brighter day
Some might say we will find a brighter day, yeah

'Cause I've been standing at the station
In need of education in the rain
You made no preparation for my reputation once again
The sink is full of fishes
'Cause she's got dirty dishes on the brain
And my dog's been itchin'
Itchin' in the kitchen once again

Some might say (some might say)
Some might say (some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
You know what some might say (you know what some might say)
(Woo!)
 Comment:
```

**Comment by deluan** [comment]
Sorry, I haven't try with that change, as I thought that PR was just for image tags... I'll do more tests, thanks!

**Comment by deluan** [comment]
You are right, this will be fixed once #69 is merged.
